classes/request.class.php

```diff
@@ -181,6 +181,9 @@ public function send_request($endpoint, $requestbody, $method, $requesttype = 'g
         }
 
         $result = curl_exec($ch);
+        if ($result === false) {
+            $this->logger->error('Curl error: ' . curl_error($ch));
```

Comment:
Should we have something more descriptive than "Curl error"? Or do you think that's sufficient for this? Will this log the URL being called, the payload, the response status code, etc? Or is this just if there is no response?

Reply:
This won't log those things. curl_error is specific to the curl request output. An example from when I was testing it:

`[error] Curl error: error setting certificate verify locations:
  CAfile: gffgj
  CApath: /etc/ssl/certs`

However, we do log those things in the same method already for each request. So we'll have the details about the URL being called, the payload and response code, followed by the curl error if one exists. This just builds on top of that. I just set it as "Curl error" because as you see above the curl error itself has a description, which is part of PHP Curl implementation. So that may be sufficient. Could probably get away with not prefixing with "Curl error" but I think stating it comes from Curl is important for people not familiar with this stuff as from "error setting certificate verify locations" on its own for example isn't obvious it's a Curl problem.